src/components/Article.jsx

```diff
@@ -1 +1,34 @@
-export default function Article() {}
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { articleById } from "../api";
+import ArticleStyle from "./ArticleStyle";
+
+export default function Article() {
+  const [article, setArticle] = useState();
+  const { article_id } = useParams({});
+  const [isloading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    articleById(article_id).then(({ data }) => {
+      setArticle(data.article);
+      setIsLoading(false);
```

Comment:
Nice loading pattern! On a similar vein - what would happen if someone came to look for an article of 99999 or schmarticle?